src/reducers/riderStageConnectionReducer.js

```diff
@@ -0,0 +1,20 @@
+import * as types from "../actions/actionTypes";
+
+const initialState = {
+  cons: [],
+  error: false
+};
+
+const riderStageConnectionReducer = (state = initialState, action) => {
+  switch(action.type) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement with "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1301)